packages/notifications/src/styled/StyledWell.ts

```diff
@@ -5,32 +5,30 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import styled from 'styled-components';
-import classNames from 'classnames';
-import CalloutStyles from '@zendeskgarden/css-callouts';
-import { retrieveComponentStyles, isRtl } from '@zendeskgarden/react-theming';
+import styled, { css } from 'styled-components';
+import { getColor, retrieveComponentStyles } from '@zendeskgarden/react-theming';
+import { StyledBase } from './StyledBase';
 
 const COMPONENT_ID = 'notifications.well';
 
 export interface IStyledWellProps {
   isRecessed?: boolean;
-  isFloating?: boolean;
 }
 
 /**
  * Supports all `<div>` props
  */
-export const StyledWell = styled.div.attrs<IStyledWellProps>(props => ({
+export const StyledWell = styled(StyledBase).attrs({
   'data-garden-id': COMPONENT_ID,
-  'data-garden-version': PACKAGE_VERSION,
-  className: classNames(CalloutStyles['c-callout'], {
-    // RTL
-    [CalloutStyles['is-rtl']]: isRtl(props),
+  'data-garden-version': PACKAGE_VERSION
+})<IStyledWellProps>(props => {
+  const { theme } = props;
```

Comment:
We don't desctructure theme/space like this in other components. Could we align this with our other usages?

Reply:
After that ☝️ this can all just become a straight template string without any `css` return.

Reply:
Yeah aligning with other usages sound good. Fixed 👍 .